test/03_before_build/version.txt

```diff
@@ -1,2 +0,0 @@
-3.6.0 (default, Feb  7 2017, 23:55:32) 
```

Comment:
What was this file doing? I don't see this in the new tests, anymore?

Reply:
I don't know what it was doing there! It must have been committed by mistake 😳